flexmeasures/data/services/sensors.py

```diff
@@ -482,7 +482,9 @@ def _get_sensor_stats(
     # build main query
     q = (
         sa.select(
-            DataSource.name,
+            sa.func.concat(
+                DataSource.name, "-", DataSource.model, "-", DataSource.version
+            ).label("combined_name_model_version"),
```

Comment:
That doesn't look very nice for sources without a model or version, though. What if you get the object and do source.description, instead?